<?xml version="1.0"?>

<launch>


      <node pkg="rosbag" type="play" name="rosbag" required="true" args="/home/mark/rosBags/June_11/test1.bag"/>
    <!--   <node name="extract" pkg="image_view" type="video_recorder" respawn="false" required="true" output="screen" cwd="ROS_HOME">
        <remap from="image" to="/tracks_video/image_raw"/>
        <param name="fps" value="60"/>
        <param name="encoding" value="bgr8"/>
      </node>
     -->



     <node name="video_extraction" pkg="video_extraction" type="video_extraction_node" output="screen" cwd="ROS_HOME">
       <remap from="image" to="/tracks_video/image_raw"/>
     </node>
     
     


</launch>